coremltools/converters/mil/mil/ops/defs/iOS16/constexpr_ops.py

```diff
@@ -141,8 +138,8 @@ def rank_promoted_to_same_as_quantized_data(param):
 class constexpr_cast(Operation):
     """
     A compile-time operation that returns a constant output value upon casting its constant input.
-
-    Expression: output = constexpr_cast(source_val, output_dtype="fp32")
+    ::
```

Comment:
Just curious, what's the point of the double colons?

Reply:
The double-colons (`::`) designate an indented literal block of text in ReStructured Text (RST), the format for docstrings. No markup processing is done within a literal block. It is left as-is, and is typically rendered in a monospaced typeface. (I learned all this from [reStructuredText Markup Specification](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html).)